tutorials/wpa-enterprise.rst

```diff
@@ -0,0 +1,373 @@
+FreeRADIUS for WPA Enterprise (EAP-TTLS-PAP)" section in the
+ansible-openwisp2 documentation
+<https://github.com/openwisp/ansible-openwisp2/tree/master?tab=readme-ov-file#configuring-freeradius-for-wpa-enterprise-eap-ttls-pap>`_
+for details.
+
+If you still prefer to configure the FreeRADIUS site manually, you can
+refer the `"Freeradius Setup for WPA Enterprise (EAP-TTLS-PAP)
+authentication" section of the OpenWISP RADIUS documentation
+<https://openwisp-radius.readthedocs.io/en/stable/developer/freeradius_wpa_enterprise.html#freeradius-setup-for-wpa-enterprise-eap-ttls-pap-authentication>`_.
+
+Creating the NAS
+----------------
+
+.. note::
+
+    You can skip this step if you are following this tutorial on our
+    :doc:`Demo System <./demo>`. The NAS has been already configured on
+    the :doc:`Demo System <./demo>`.
+
+From the OpenWISP navigation menu, go to ``RADIUS`` and then ``NAS``, from
+here click on the ``Add NAS``.
+
+.. image:: ../images/navigating-to-nas.png
+    :target: ../_images/navigating-to-nas.png
+
+Fill in the organization, short name, secret, and set the type to
+"Wireless - IEEE 802.11". In the **name** field, enter the IP address of
+the NAS. Since every device acts as a NAS in our scenario, we specify the
+subnet of the VPN. This would allow FreeRADIUS to accept RADIUS traffic
+from all the devices.
+
+.. image:: ../images/wpa-enterprise/create-nas.png
+    :target: ../_images/create-nas.png
+
+.. warning::
+
+    Creating or modifying a NAS in OpenWISP requires a restart of the
+    FreeRADIUS server. Otherwise, the changes won't take effect.
+
+We would need the NAS's secret in the next step while creating a template.
+
+Creating the Template
+---------------------
+
+.. note::
+
+    This template is also available in our :doc:`Demo System <./demo>` as
+    `WPA Enterprise (EAP-TTLS)
+    <https://demo.openwisp.io/admin/config/template/5f279920-60fd-4274-b367-450aa4d30004/change/>`_,
+    **feel free to try it out!**
+
+From the OpenWISP navigation menu, go to ``Configurations`` and then
+``Templates``, from here click on the ``Add template``.
+
+.. image:: ../images/create-template.png
+    :target: ../_images/create-template.png
+
+Fill in name, organization, leave type set to "Generic", backend set to
+"OpenWrt", scroll down to the Configuration section, then click on
+"Advanced mode (raw JSON)".
+
+.. image:: ../images/advanced-mode.png
+    :target: ../_images/advanced-mode.png
+
+Before copying the following NetJSON to the advanced mode editor, you will
+need to update these fields to reflect your configuration:
+
+- ``key`` - RADIUS secret should be same as set in NAS
+- ``server`` - RADIUS server authentication IP
+- ``port`` - RADIUS server authentication port
+- ``acct_server`` - RADIUS accounting server IP
+- ``acct_server_port`` - RADIUS accounting server port
+
+.. code-block:: json
+
+    {
+        "interfaces": [
+            {
+                "wireless": {
+                    "network": [
```

Comment:
Please reorder this JSON so that `name` and `type` come first, the `wireless` property should be last.